src/django_otp/plugins/otp_email/models.py

```diff
@@ -106,7 +107,7 @@ def _deliver_token(self, extra_context):
             html_message=body_html,
         )
 
-        message = "sent by email"
+        message = str(_("sent by email")).capitalize()
```

Comment:
or maybe just `message = _("sent by email")` :smile: ?

Reply:
> or maybe just `message = _("sent by email")` 😄 ?

Yes, usually best to keep it simple.